Podfile.lock

```diff
@@ -1,16 +1,28 @@
 PODS:
+  - FutureNova (0.1.5)
   - SnapKit (5.0.1)
 
 DEPENDENCIES:
+  - FutureNova (from `https://github.com/cuappdev/ios-networking.git`)
   - SnapKit
 
 SPEC REPOS:
-  trunk:
+  https://github.com/cocoapods/specs.git:
     - SnapKit
 
+EXTERNAL SOURCES:
+  FutureNova:
+    :git: https://github.com/cuappdev/ios-networking.git
+
+CHECKOUT OPTIONS:
+  FutureNova:
+    :commit: fdbf935dab64617ac2353a720d7cb8fd71b67327
+    :git: https://github.com/cuappdev/ios-networking.git
+
 SPEC CHECKSUMS:
+  FutureNova: 6a0ed485ee7e4d6606eb9b5b33e0f42a5cd80726
   SnapKit: 97b92857e3df3a0c71833cce143274bf6ef8e5eb
 
-PODFILE CHECKSUM: 107a7d8cb033c1b3563ab4c80c137395fac512f9
+PODFILE CHECKSUM: df7f8a417dea5fb6d84a3eb59a2a517a2b123a48
 
-COCOAPODS: 1.8.3
+COCOAPODS: 1.7.5
```

Comment:
update your cocapods (`sudo gem install cocoapods`) and then rerun `pod install`!